docs/src/templates/includes/components/core/je-restaurant-listing--delivery.hbs

```diff
@@ -0,0 +1,33 @@
+<section class="c-listing-item" itemscope="" itemtype="http://schema.org/Restaurant" data-restaurant-id="123" data-test-id="123">
+    <a href="#" class="c-listing-item-link u-clearfix" itemscope="" itemtype="http://schema.org/Restaurant"
+       title="View the Menu for Fozzie Kebab">
+        <div class="c-listing-item-img">
+            <img class="js-lazy c-listing-item-imgItem" data-ft="restaurantDetailsLogo" src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" />
+            <noscript>img class="c-restaurant__logo js-lazy" data-ft="restaurantDetailsLogo" src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" itemprop="image";</noscript>
```

Comment:
I think the formatting in the `noscript` sections in this PR are a little broken, should be something like this:

```
<img class="c-listing-item-imgItem" src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" itemprop="image" />
```